cumulus/ansible/tasks/volume.py

```diff
@@ -0,0 +1,137 @@
+from cumulus.celery import command
+from cumulus.ansible.tasks.inventory import simple_inventory
+
+from cumulus.ansible.tasks.utils import run_playbook
+from cumulus.ansible.tasks.utils import get_playbook_directory
+from cumulus.ansible.tasks.utils import get_callback_plugins_path
+from cumulus.ansible.tasks.utils import get_library_path
+
+import os
+from cumulus.ssh.tasks.key import _key_path
+
+
+@command.task
+def create_volume(profile, volume, secret_key, girder_callback_info):
+
+    playbook = os.path.join(get_playbook_directory(),
+                            'volumes', 'ec2', 'create.yml')
+
+    extra_vars = {
+        'girder_volume_id': volume['_id'],
+        'profile_id': profile['_id'],
+        'volume_name': volume['name'],
+        'volume_size': volume['size'],
+        'volume_zone': volume['zone'],
```

Comment:
Doesn't the zone have to match our profile? i.e. if our access key is in us-east-2 we can't create a volume in us-west-2?